roles/reproducer/tasks/generate_deploy_values.yml

```diff
@@ -1 +0,0 @@
----
```

Comment:
is this file empty now? does it have any use?

Reply:
it's removed in fact :)